pkgs/tools/audio/beets/common.nix

```diff
@@ -158,6 +161,8 @@ python3Packages.buildPythonApplication {
     "test/plugins/test_autobpm.py::TestAutoBPMPlugin::test_import"
     # AssertionError: assert 0 == 117
     "test/plugins/test_autobpm.py::TestAutoBPMPlugin::test_command"
+    # AssertionError: Image written is not /build/source/test/rsrc/abbey-similar.jpg
+    "test_accept_similar_art"
```

Comment:
Could we separate this build failure fix to a separate commit? Also note https://github.com/beetbox/beets/pull/5650 and https://github.com/NixOS/nixpkgs/issues/385389 . I'd like to wait a day or so for upstream to merge that PR.